### ISSUE TITLE
www does not have attributes

### ISSUE THREAD

**Issue by averlon** [FriendsOfTYPO3/tt_address#136]
## Bug Report

**Current Behavior**
I have created a partial to setup the address in my frontent and also included the www.

`
    <f:if condition="{address.www}">
        <meta itemprop="url" content="{address.www}"/>
        <li>
            <i class="fas fa-link"></i>
            {f:translate(key:'label.www')}:
            <f:link.external uri="{address.www}">{address.www}</f:link.external>
        </li>
    </f:if>
`
when tracing it it is shown as:

`www =>protected'http://www.forstinning.de _blank - "Gemeinde"' (58 chars)`

This is, if I am not wrong, the complete string created in the T3 backend via the T3 form for Links als External URL.

**Expected behavior/output**
I would expect the string in the content to be split up in "uri", "target" and, I guess, "alt".

Probably my f:link is wrong.

**Environment**
- TYPO3 version(s): 9.5.3
- tt_address version: 4.0.1
- Is your TYPO3 installation set up with Composer (Composer Mode): no
- OS: Ubuntu 18.04

**Possible Solution**

**Additional context**
If my f.link is wrong I would appreciate a hint how to set it correct.

Thanks
Kallewirsch

**Comment by bmack** [comment]
Hey,

could you try to use the `<f:link.typolink>` viewhelper instead?

**Comment by averlon** [comment]
Hi,
if I am not wrong, the "f:link.external ..." came with your partials you delivered via the extensions.

You deliver the Address.html as partial and this contains this setting. That's why I took it.

I will give it a try with typolink but I am not sure if this will work since there are some information missing typolink would need - I will check.

Please check your standard partials as well!

Regards
Kallewirsch

**Comment by averlon** [comment]
Hi,
in the latest release of tt_address you delivered a new partial with the following code:
`        <f:if condition="{address.www}">
            <meta itemprop="url" content="{f:uri.typolink(parameter:address.www)}"/>
            <li>
                <i class="fas fa-link"></i>
                {f:translate(key:'label.www')}:
                <f:link.typolink parameter="{address.www}">{address.www}</f:link.typolink>
            </li>
        </f:if>
`

This still brings result:
` http://www.xxxx.de _blank`

Has there something else to be done in the partial to have the "target" and also the "title" separated from the link text shown in the frontend?

**Comment by averlon** [comment]
Hi Georg,
this now works - thanks. The hint to "address.wwwSimplified" was not visible to me. Neither in the comment of the issue, nor in the partial delivered with the latest release.
Probably you could update the partial again in your next release.
Thanks
Kallewirsch

**Comment by georgringer** [comment]
But this is added in latest partial, isn't it?

**Comment by averlon** [comment]
Hi Georg,
no, I don't think so.

This is the source I can finde in the partial "Address.html".

`        <f:if condition="{address.www}">
            <meta itemprop="url" content="{f:uri.typolink(parameter:address.www)}"/>
            <li>
                <i class="fas fa-link"></i>
                {f:translate(key:'label.www')}:
                <f:link.typolink parameter="{address.www}">{address.www}</f:link.typolink>
            </li>
        </f:if>`

**Comment by georgringer** [comment]
Don't get it, in current master the code is

<f:link.typolink parameter="{address.www}">{address.wwwSimplified}</f:link.typolink>

Where is the code you mention?

**Comment by averlon** [comment]
Hi Georg,
I only take the code from the latest update of extensions in my installation, not from github.
The version I have installed is 4.1.0. And I took a look at typo3conf\ext\tt_address\Resources\Private\Partials\

I normally rely on the delivery of updates. Up to now I have never taken something from github to update my installation.

But anyhow. Maybe I have looked at the wrong place. If it is now in the sources - all is fine.

Thanks for the Extension!

P.S.: If you want to have a look : http://www.averlon.de/av-wo/av-loc/av-vereine
